js/main.js

```diff
@@ -21,5 +21,83 @@ function submitForm(event) {
   data.entries.unshift(formObject);
   $img.setAttribute('src', 'images/placeholder-image-square.jpg');
   $submit.reset();
+  document.getElementById('ul').prepend(renderEntry(formObject));
```

Comment:
`querySelector`s or `getElementById` in this case, should be done outside the function. You don't want to do this inside the function, because you're forcing us to traverse the DOM over and over again when the function gets called, rather than just doing this globally and doing the search one time